cpp/src/Ice/ios/StreamEndpointI.h

```diff
@@ -35,7 +35,8 @@ class Instance : public IceInternal::ProtocolInstance
 public:
 
     Instance(const Ice::CommunicatorPtr&, Ice::Short, const std::string&, bool);
-    virtual ~Instance();
+    Instance(const Instance&, const IceInternal::ProtocolInstancePtr&);
```

Comment:
I would change `Instance&` to `InstancePtr&` and replace `*_instance.get()` with `_instance`

Reply:
I was trying to avoid having to forward declare Instance. I guess it could take `shared_ptr<Instance>`?

Reply:
I added the forwards decl.